source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -118,6 +122,9 @@ export class Carousel extends Container {
 
     private _pages: CarouselPage[] = [];
     private _renderedPages: CarouselPage[];
+    private _carouselPageContainer: HTMLElement;
+
+    currentIndex: number = 0;
```

Comment:
Is it `private`? or add a get function with private _currentIndex?
Is it only be used in designer mode, possible a name like currentIndexInDesignerMode

Reply:
We also use currentIndex within `SwiperOptions` to assign the carousel's initial index, so I'm not sure renaming it makes sense.

We re-render the carousel whenever we switch pages in the designer, so this makes sure we save the state.
The default is 0 so it shouldn't affect the carousel in general, but it is a good start if we ever want to add `currentPageIndex` to the schema.